### DISCUSSION TITLE
Lion making training on 12Gb GPU difficult

### DISCUSSION THREAD

**Discussion by Steppr** [d8ahazard/sd_dreambooth_extension#1038] [General]
I wanted to try the Lion Scheduler for training. I found that I cannot start a Training session with Lion enabled due to running out of Memory on my RTX 4070 Ti. 
I have disabled all the settings known to increase Vram usage except Text Encoder training because that's appearently very important for good results. Disabling Lion and using 8bit adam instead with the exact same settings (except learning rate * 10) works and is using 10.8 GB of Vram. 
Am I the only one with that experience? Has anybody else played around with it? Perhaps it should go onto the Vram hungry settings list if it is indeed eating lots of vram.

**Comment by 78Alpha** [comment]
I gave it a minute test to see what it did and it does increase memory usage. Also decreased the speed as a byproduct of batch size 64 getting dropped to 32 by accelerate.

For a few hundred steps it goes smoothly, but when the opportunity kicks in it can try to grab 2 or even 4 GB VRAM.
